.github/workflows/deploy-website.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
         working-directory: ./website
         # XXX: Drop `--disable_external true` once we fully adopted the
         # "Refaster rules" terminology on our website and in the code.
-        run: bundle exec htmlproofer --disable_external true --check-external-hash false ./_site
```

Comment:
Execution of `htmlproofer` will eventually live in a Ruby script instead.

Reply:
The `favicon.ico` should be in the root for best browser support, sorry 😕 The error was actually correct by `htmlproofer` 😉 

See here for more details https://realfavicongenerator.net/faq#why_icons_in_root